packages/wallet/src/views/Home.jsx

```diff
@@ -80,29 +81,46 @@ const memberCards = [
   {
     title: 'UnProfile',
     description: 'Unprofile yourself',
-    url: `${env === 'STAGING' ? 'https://staging.unprofile.kernel.community/' : 'https://unprofile.kernel.community'}`,
+    url: `https://${prefix}unprofile.kernel.community`,
     active: true
   },
   {
     title: 'Adventure',
     description: 'Heed the call to adventure',
-    url: `${env === 'STAGING' ? 'https://staging.adventures.kernel.community/' : 'https://adventures.kernel.community'}`,
+    url: `https://${prefix}adventures.kernel.community`,
     active: true
   },
   {
     title: 'Explore',
     description: 'Connect with other Fellows',
-    url: `${env === 'STAGING' ? 'https://staging.explore.kernel.community/' : 'https://explore.kernel.community'}`,
+    url: `https://${prefix}explore.kernel.community`,
     active: true
   },
   {
     title: 'Chat',
     description: 'Horizontal conversations',
-    url: `${env === 'STAGING' ? 'https://staging.chat.kernel.community/' : 'https://chat.kernel.community'}`,
+    url: `https://${prefix}chat.kernel.community`,
     active: false
   }
 ]
 
+const LinkCard = ({ cardConfig }) => {
+  const Card = cardConfig.map((card, index) => {
+    return (
+      <a key={index} href={`${card.active ? card.url : '#'}`}>
+        <div className={
+          `${card.active ? 'bg-kernel-dark text-kernel-white' : 'bg-kernel-grey'} p-5 rounded shadow-md`
+        }
+        >
+          <div className='text-xl mb-2'>{card.title}</div>
+          <div className='text-base mb-1'>{card.description}</div>
+        </div>
+      </a>
+    )
+  })
+  return Card
+}
+
```

Comment:
1. It's helpful to name something in the plural when it refers to a collection of things, so maybe `LinksCard` or `LinkCards` depending on how you're thinking about it.
2. `const Card` is unnecessary; you can just `return cardConfig.map((...` and remove line 121.